ui/store/actions.ts

```diff
@@ -2638,6 +2638,16 @@ export function hideLoadingIndication(): Action {
   };
 }
 
+/**
+ * An action creator for display a warning to the user in various places in the
+ * UI. It will not be cleared until a new warning replaces it or `hideWarning`
+ * is called.
+ *
+ * @deprecated This way of displaying a warning is confusing for users and
+ * should no longer be used.
+ * @param text - The warning text to show.
```

Comment:
Also maybe better to update how this is described, and use the same parameter name
```suggestion
 * @param payload - The warning to show.
```